packages/devtools_app/test/version_test.dart

```diff
@@ -3,8 +3,6 @@
 // found in the LICENSE file.
 
 // Ignoring redundant argument values makes the test easier to read.
```

Comment:
Since we removed the ignore, remove this comment that goes with it as well.

Reply:
done in https://github.com/flutter/devtools/pull/4011